src/components/graph/menus/network-modifications/node-editor.tsx

```diff
@@ -16,7 +16,7 @@ import { Alert, Box } from '@mui/material';
 import { AppState } from '../../../../redux/reducer';
 import { CheckCircleOutlined } from '@mui/icons-material';
 import { FormattedMessage } from 'react-intl';
-import ComputingType from 'components/computing-status/computing-type';
+import { ComputingType } from '@gridsuite/commons-ui';
```

Comment:
line 49 : use `selectLoadflowComputingStatus` as well here ? check for other occurences